### PR TITLE
SW-4728 Remove project filter if there are no projects within the accessions view

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2097?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #2096** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2096?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #2097** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2097?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
      * **PR #2098** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2098?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2097?utm_source=stack-comment).

File: src/components/seeds/database/index.tsx [hunk 1]
```diff
@@ -452,13 +461,21 @@ export default function Database(props: DatabaseProps): JSX.Element {
...
-          const allValues = await SeedBankService.searchFieldValues(
-            singleAndMultiChoiceFields,
-            {},
-            selectedOrganization.id
+          // Since the seed bank service doesn't return values for project IDs, we need to merge the values in
+          const singleAndMultiChoiceFields = filterSelectFields(
+            searchColumns.filter((column) => column !== 'project_name')
           );
 
+          const allValues =
```
Comment (karthikbtf) [inline src/components/seeds/database/index.tsx:469]: this can be null, add a guard around line 473?

**Review by karthikbtf — APPROVED** [review]

File: src/components/seeds/database/index.tsx (hunk 1 above)
Comment (nickgraz) [reply src/components/seeds/database/index.tsx:469]: I am changing it to `{}` if it is `null` on the next line. Would you rather I leave it and check for `null` or is it OK as-is? Currently `allValues` will never be `null`.

Comment (karthikbtf) [reply src/components/seeds/database/index.tsx:469]: I missed the `|| {}` my bad.. this is totally fine.

**Comment by nickgraz** [comment]
### Merge activity

* **Jan 26, 5:35 PM**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2097).

**Pull request merged**